tests/unit/modules/network/vyos/test_vyos_bgp_address_family.py

```diff
@@ -367,8 +381,9 @@ def test_vyos_bgp_address_family_overridden(self):
             "delete protocols bgp 65536 neighbor 203.0.113.5 address-family",
             "delete protocols bgp 65536 neighbor 192.0.2.25 address-family",
             "delete protocols bgp 65536 address-family ipv6-unicast redistribute ripng",
+            "delete protocols bgp 65536 address-family ipv4 redistribute",
             "delete protocols bgp 65536 address-family ipv4 aggregate-address",
-            "delete protocols bgp 65536 address-family ipv4-unicast network 192.2.13.0/24",
+            "delete protocols bgp 65536 address-family ipv4-unicast network 192.2.13.0/24 backdoor",
```

Comment:
I had to have this pair of commands even though the real test with VM generate only a single one "delete protocols bgp 65536 address-family ipv4-unicast network 192.2.13.0/24". I did not got to the bottom of why test suite behaves differently and wants two delete commands

Reply:
This is addressed in PR#367